pages/levels/6.vue

```diff
@@ -0,0 +1,128 @@
+<script lang="ts">
+import { defineComponent } from "vue";
+import Level from "~/components/Level.vue";
+
+/*
+
+
+
+
+
+*/
+
+const code = `    <p class="result"></p>`;
+
+export default defineComponent({
+  data() {
+    return {
+      markdown: `There are some parts of a website, which can't be made accessible by simply using native HTML elements, which you learned about in the previous level. For example, look at this small application for throwing dice digitally. After clicking on the button, screen reader wouldn't be notified that something has changed. They would have manually search the part of the website that has changed. Not very user-friendly.
```

Comment:
...parts of a website *that* can't be... Klingt so besser, finde ich

...native HTML *elements, which you learned about in the previous level*. Klingt etwas verwirrend, ich würde es anders formulieren, was besseres fällt mir im Moment leider auch nicht ein

...search*ed*.../ ...have *had to* manually search...